DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/mmhost/part3_hosts/src/mmhost.cpp

```diff
@@ -1,10 +1,11 @@
 #include <sycl/ext/intel/fpga_extensions.hpp>
 #include <sycl/sycl.hpp>
 
-#include "exception_handler.hpp"
+// #include "exception_handler.hpp"
```

Comment:
```suggestion
#include "exception_handler.hpp"
```

other people in the code samples committee like this exception handler. I don't personally care for it, but make sure you use it in these code samples

Reply:
Changed the CMake files to actually include that file, no need to comment it out anymore